admin/extensions.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace HM\BackUpWordPress;
+
+?>
+
+<div class="wrap">
+
+	<h1>
+
+		<a class="page-title-action" href="<?php echo get_settings_url() ?>"><?php _e( '&larr; Backups', 'backupwordpress' ); ?></a>
+
+		<?php _e( 'BackUpWordPress Extensions', 'backupwordpress' ); ?>
+
+	</h1>
+
+	<div class="wp-filter">
+		<p><?php _e( 'Extend BackUpWordPress by installing extensions. Extensions allows you to pick and choose the exact features you need whilst also supporting us, the developers, so we can continue working on BackUpWordPress.', 'backupwordpress' ); ?></p>
+	</div>
+
+	<?php
+	$extensions_data = Extensions::get_instance()->get_edd_data();
+
+	// Sort by title
+	usort( $extensions_data, function( $a, $b ) {
+		return strcmp( $b->title->rendered, $a->title->rendered );
+	});
+
+	$installed_plugins = array_map( 'strtolower' , wp_list_pluck( get_plugins(), 'Name' ) );
+	?>
+
+	<h3><?php _e( 'Remote Storage', 'backupwordpress' ); ?></h3>
+
+	<p><?php _e( 'It\'s important to store your backups somewhere other than on your site. Using the extensions below you can easily push your backups to one or more Cloud providers.', 'backupwordpress' ); ?></p>
+
+	<div class="wp-list-table widefat plugin-install">
+		<div id="the-list">
+			<?php $first = true; ?>
+			<?php foreach ( $extensions_data as $extension ) : ?>
+
+				<div class="plugin-card plugin-card-<?php echo esc_attr( $extension->slug ); ?>">
+
+					<div class="plugin-card-top">
+
+						<div class="name column-name">
+							<h4>
+								<a href="<?php echo esc_url( $extension->link ); ?>" class="thickbox">
+									<?php echo esc_html( $extension->title->rendered ); ?>
+									<img src="<?php echo esc_url( $extension->featured_image_url ); ?>" class="plugin-icon" alt="">
+								</a>
+							</h4>
+						</div>
+
+						<div class="action-links">
+							<ul class="plugin-action-buttons">
+								<li>
+									<?php if ( in_array( strtolower( $extension->title->rendered ), $installed_plugins ) ) : ?>
+										<span class="button button-disabled" title="<?php _e( 'This extension is already installed and is up to date', 'backupwordpress' ); ?>"><?php _e( 'Installed', 'backupwordpress' ); ?></span>
```

Comment:
<img width="543" alt="screenshot 2016-03-15 14 20 11" src="https://cloud.githubusercontent.com/assets/30460/13780872/0ec77d6c-eab9-11e5-80f9-5b9ce77a847d.png">

Now we detect if an update is required and link to the updates page